src/Runner.Worker/ExecutionContext.cs

```diff
@@ -813,6 +813,12 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
                 Environment.SetEnvironmentVariable(Constants.Variables.Agent.ForcedInternalNodeVersion, "node16");
             }
 
+            var forceActionsToNode20 = StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable(Constants.Variables.Agent.ManualForceActionsToNode20));
```

Comment:
Curious why we need an env var to control this? Is this for customers to have a way to test their changes?

Reply:
Curious because this is checking machine process variables here, not `vars` or `env` from the workflow context

Wondering whether we expect customers to set an environment variable on their machines? Customers who use hosted runners will not have a way to do this.

Reply:
Ops, I missed checking workflow variable here. Fixed